app/talk/discussion.cjsx

```diff
@@ -49,6 +46,18 @@ module.exports = React.createClass
     if @props.params.discussion isnt nextProps.params.discussion
       @setDiscussion(nextProps.params.discussion)
         .then => @setComments(nextProps.location.query.page ? 1)
+        .then =>
+          if nextProps.user isnt @props.user
+            talkClient
```

Comment:
I'm not sure. I copied this straight from the coffeescript without really thinking about it.